service/frontend/service.go

```diff
@@ -293,6 +296,7 @@ func NewConfig(
 		AdminDeleteAccessHistoryFraction: dc.GetFloat64Property(dynamicconfig.FrontendAdminDeleteAccessHistoryFraction, 0.0),
 
 		EnableNexusHTTPHandler:      dc.GetBoolProperty(dynamicconfig.FrontendEnableNexusHTTPHandler, false),
+		EnableOpenAPIHTTPHandler:    dc.GetBoolProperty(dynamicconfig.FrontendEnableOpenAPIHTTPHandler, false),
```

Comment:
Do we really need a dynamic config here? If we do, can we make it on by default so we don't need to clutter up the development config files and test setup? Why would anyone want to turn this off, anyway?

Reply:
It was a precaution in case we didn't wish to provide docs while we reviewed them, but the whole HTTP API isn't release-ready anyways so we don't need it